addons/mail/static/src/composer/composer.js

```diff
@@ -20,6 +21,10 @@ export class Composer extends Component {
             (focus) => {
                 if (focus && this.ref.el) {
                     this.ref.el.focus();
+                    if(this.caretLocationBeforeAddingEmoji) {
+                        this.ref.el.setSelectionRange(this.caretLocationBeforeAddingEmoji, this.caretLocationBeforeAddingEmoji);
+                        this.caretLocationBeforeAddingEmoji = null;
+                    }
```

Comment:
Linter/prettier not happy with these LOCs,

Make sure to configure them properly on your machine. See `@web/tooling` (notably `enable.sh`)